src/penn_chime/models.py

```diff
@@ -269,13 +269,18 @@ def sir(
     s_n = (-beta * s * i) + s
```

Comment:
I think the right way to say this is

```python
new_infections = min(s, beta*s*i)
s_n = s - new_infections
i_n = i + new_infections - gamma*i
```

And similar for recoveries

Reply:
Justification: there are only `s` people left to become infected, and there's conservation of people across pools.

Reply:
> conservation of people across pools.

one important property test i'd recommend is at every time step `assert s + i + r == p.population` or something

Reply:
@PhilMiller I can't verify your algebra. I did half the cases for `s_n` by making an assumption, which went fine actually. but `i_n` was a mess, none of the four cases terminated.
![IMG_20200328_174520](https://user-images.githubusercontent.com/39039420/77834564-02e7f380-711c-11ea-875f-5a5f675c4b31.jpg)

Reply:
it reminds me that i should write a `validate internal state of model` issue.

Reply:
I added a card to check this one.